### PR TITLE
Iss25 modeling email styles

### PR DESCRIPTION
**Pull request by MiguelOcegueraM**
# Description
 - We gave styles to the emails that users who register on our platform and receive notifications through email will receive
 
 # Testing
- Clone our repo
- Start the server with rails s
- Generate a mail for you
- Go to http://localhost:3000/to see the welcome page

  ## Screenshots 
Screenshot_2021-04-13-18-33-26-875_com google android gm
Screenshot_2021-04-13-18-33-26-875_com google android gm

### CODE REVIEW

File: config/initializers/warden/proxy.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# module Warden
```
Comment (OchoaHub) [inline config/initializers/warden/proxy.rb:1]: ¿?

File: app/views/user_mailer/send_alert.html.erb [hunk 2]
```diff
@@ -2,26 +2,83 @@
...
   </head>
-  <body>
-    <h1>Incident alert!!!</h1>
+    <body style="margin: 0; padding: 0;">
+    <table role="presentation" border="0" cellpadding="0" cellspacing="0" width="100%">
+      <tr>
+        <td style="padding: 20px 0 30px 0;">
 
-    <h3>Incidents were detected in the following places:<h3>
+  <table align="center" border="0" cellpadding="0" cellspacing="0" width="600" style="border-collapse: collapse; border: 1px solid #cccccc;">
```
Comment (OchoaHub) [inline app/views/user_mailer/send_alert.html.erb:12]: Use ruby-beautify gem for this code :)

File: app/views/user_mailer/send_alert.html.erb [hunk 3]
```diff
@@ -2,26 +2,83 @@
...
+          </ul>
+        
+
+        </table>
+      </td>
+    </tr>
+    <tr>
+      <td bgcolor="#ee4c50" style="padding: 30px 30px;">
+          <table border="0" cellpadding="0" cellspacing="0" width="100%" style="border-collapse: collapse;">
+          <tr>
```
Comment (OchoaHub) [inline app/views/user_mailer/send_alert.html.erb:50]: Use ruby-beautify for this code :)

File: app/views/user_mailer/send_key.html.erb [hunk 4]
```diff
@@ -3,19 +3,78 @@
...
-    <p>
-      We appreciate your preference.
+      <body style="margin: 0; padding: 0;">
+    <table role="presentation" border="0" cellpadding="0" cellspacing="0" width="100%">
+      <tr>
+        <td style="padding: 20px 0 30px 0;">
 
-      Here is your access key. Enjoy and protect yourself.
+  <table align="center" border="0" cellpadding="0" cellspacing="0" width="600" style="border-collapse: collapse; border: 1px solid #cccccc;">
+    <tr>
```
Comment (OchoaHub) [inline app/views/user_mailer/send_key.html.erb:12]: Use ruby-beautify for this code :)

File: app/views/user_mailer/send_key.html.erb [hunk 5]
```diff
@@ -3,19 +3,78 @@
...
-    <h1>Dear user</h1>
-    <p>
-      We appreciate your preference.
+      <body style="margin: 0; padding: 0;">
+    <table role="presentation" border="0" cellpadding="0" cellspacing="0" width="100%">
+      <tr>
+        <td style="padding: 20px 0 30px 0;">
 
-      Here is your access key. Enjoy and protect yourself.
+  <table align="center" border="0" cellpadding="0" cellspacing="0" width="600" style="border-collapse: collapse; border: 1px solid #cccccc;">
```
Comment (OchoaHub) [inline app/views/user_mailer/send_key.html.erb:11]: CSS in HTML?? I think it's not a good practice, be careful

File: app/views/user_mailer/send_key.html.erb [hunk 6]
```diff
@@ -3,19 +3,78 @@
...
+            </td>
+            <td align="right">
+              <table border="0" cellpadding="0" cellspacing="0" style="border-collapse: collapse;">
+                <tr>
+                  <td>
+                    <a href="http://www.twitter.com/">
+                      <img src="https://assets.codepen.io/210284/tw.gif" alt="Twitter." width="38" height="38" style="display: block;" border="0" />
+                    </a>
+                  </td>
+                  <td style="font-size: 0; line-height: 0;" width="20">&nbsp;</td>
```
Comment (OchoaHub) [inline app/views/user_mailer/send_key.html.erb:61]: Use ruby-beautify for this code :)

File: app/views/user_mailer/send_key.html.erb [hunk 7]
```diff
@@ -3,19 +3,78 @@
...
+        
 
-      USER: <%= @user[:email]%>
-      PASSWORD <%=@user[:password]%>
+        </table>
+      </td>
+    </tr>
+    <tr>
+      <td bgcolor="#ee4c50" style="padding: 30px 30px;">
+          <table border="0" cellpadding="0" cellspacing="0" width="100%" style="border-collapse: collapse;">
```
Comment (OchoaHub) [inline app/views/user_mailer/send_key.html.erb:47]: CSS in HTML?? I think it's not a good practice, be careful

**Review by OchoaHub — APPROVED** [review]
The functionality looks good

**Review by LDVera — APPROVED** [review]
looks good only use your sass files to config the styles 💯

File: app/views/user_mailer/send_key.html.erb (hunk 5 above)
Comment (MiguelOcegueraM) [reply app/views/user_mailer/send_key.html.erb:11]: It wasn't under my control bro

File: config/initializers/warden/proxy.rb (hunk 1 above)
Comment (MiguelOcegueraM) [reply config/initializers/warden/proxy.rb:1]: it's a bad practice dude!

**Review by jnegrete15 — APPROVED** [review]

**Pull request merged**